docs/jetpack-navigation.md

```diff
@@ -0,0 +1,34 @@
+# Jetpack Navigation
+
+Mavericks ships with a `mvrx-navigation` artifact that makes it easy to use nav graph scoped ViewModels.
```

Comment:
should we time merging this PR with the next release so that this is actually available once the docs are published?